src/pages/categories/[slug].js

```diff
@@ -10,9 +11,14 @@ import styles from 'styles/pages/Post.module.scss';
 export default function Category({ category, posts }) {
   const { name, description, slug } = category;
 
-  return (
-    <TemplateArchive title={name} Title={<Title title={name} />} description={description} posts={posts} slug={slug} />
-  );
+  const { metadata } = usePageMetadata({
+    metadata: {
+      ...category,
+      description: description || category.og?.description || `Read ${posts.length} posts from ${title}`,
```

Comment:
interesting, im surprised that didn't error out 🤔  even if it doesnt hit it, i would have thought since it isn't a defined variable i would throw an error. good catch!